storage/storage.go

```diff
@@ -337,25 +334,24 @@ func (storage DBStorage) GetOrgIDByClusterID(cluster types.ClusterName) (types.O
 
 func parseRuleRows(rows *sql.Rows) ([]types.RuleOnReport, error) {
 	report := make([]types.RuleOnReport, 0)
+
 	for rows.Next() {
-		var details string
-		var ruleID string
-		err := rows.Scan(&details, &ruleID)
+		var (
+			templateData string
+			ruleFQDN     types.RuleID
+			ruleKey      types.ErrorKey
```

Comment:
Maybe we should change type as well to `types.RuleKey`?

Reply:
Same answer as previous one, I was too lazy, we need to conclude on one convention(as a team) and use it everywhere.

Reply:
Ok, so do we have convention for `error_key`?